### PR TITLE
[WIP] MES-750 - e2e tweaks (MES: 781, 783, 785)

### PR DESCRIPTION
**Pull request by ammarhaiderbjss**
- Adds (optional) text to the debrief statement on the declaration-consent page, and makes it an optional checkbox.
- Makes Controlled stop tickable.
- Pass the correct manoeuvre, ECO/ETA and control stop into the left hand side of the summary.
image

### CODE REVIEW

File: src/pages/all-on-one-v2/all-on-one-v2.ts [hunk 1]
```diff
@@ -109,8 +125,19 @@ export class AllOnOneV2Page {
     return this.selectedManoeuvre === key;
   }
 
+  controlledStopTap() {
+    this.isControlledStopDone = true;
```
Comment (danielchildsbjss) [inline src/pages/all-on-one-v2/all-on-one-v2.ts]: There's no way to reverse this. Is that right?

File: src/pages/all-on-one-v2/all-on-one-v2.ts [hunk 2]
```diff
@@ -109,8 +125,19 @@ export class AllOnOneV2Page {
...
+    this.summaryMetaDataService.setControlledStopComplete();
```
Comment (danielchildsbjss) [inline src/pages/all-on-one-v2/all-on-one-v2.ts]: It looks like we're setting two things to true that mean the same thing, it's okay for now but they might become out of sync in the future and we won't know which is the correct one.

File: src/pages/all-on-one-v2/all-on-one-v2.ts (hunk 1 above)
Comment (ammarhaiderbjss) [reply src/pages/all-on-one-v2/all-on-one-v2.ts]: Yeah, it's doable but a bit of a nightmare. It conflicts with the undo'ing of serious/dangerous faults as they want to use the (click) event as well.

Comment (ammarhaiderbjss) [reply src/pages/all-on-one-v2/all-on-one-v2.ts]: Think the PO's are ok with it to be honest, I'll double check with them

Comment (danielchildsbjss) [reply src/pages/all-on-one-v2/all-on-one-v2.ts]: I think it is fine for now - there's a fair bit of inconsistency all over the DL25 and I think it all needs a bit of a holistic rethink

Comment (ammarhaiderbjss) [reply src/pages/all-on-one-v2/all-on-one-v2.ts]: Couldn't agree with you more, rethink and a rewrite!

File: src/assets/data/fault-summary-lookup.json [hunk 3]
```diff
@@ -47,5 +47,13 @@
...
-  "vehicleCheck": "Vehicle check"
+  "vehicleCheck": "Vehicle check",
+  "manoeuvreRRCtrl": "Manoeuvre - Reverse Right - Control",
+  "manoeuvreRRObs": "Manoeuvre - Reverse Right - Observation",
+  "manoeuvreRPRCtrl": "Manoeuvre - Reverse Park (Road) - Control",
+  "manoeuvreRPRObs": "Manoeuvre - Reverse Park (Road) - Observation",
+  "manoeuvreRPCCtrl": "Manoeuvre - Reverse Park (Car park) - Control",
+  "manoeuvreRPCObs": "Manoeuvre - Reverse Park (Car park) - Observation",
+  "manoeuvreFPCtrl": "Manoeuvre - Forward Park - Control",
+  "manoeuvreFPObs": "Manoeuvre - Forward Park - Observation"
```
Comment (danielchildsbjss) [inline src/assets/data/fault-summary-lookup.json]: I fixed this in develop too - I prefer your ordering so overwrite it :)

Comment (ammarhaiderbjss) [reply src/assets/data/fault-summary-lookup.json]: done :)

**Pull request merged**